spec/paperclip/media_type_spoof_detector_spec.rb

```diff
@@ -44,9 +44,18 @@
     end
   end
 
-  it "rejects a file if named .html and is as HTML, but we're told JPG" do
-    file = File.open(fixture_file("empty.html"))
-    assert Paperclip::MediaTypeSpoofDetector.using(file, "empty.html", "image/jpg").spoofed?
+  context "file named .html and is as HTML, but we're told JPG" do
+    let(:file) { File.open(fixture_file("empty.html")) }
+    let(:spoofed?) { Paperclip::MediaTypeSpoofDetector.using(file, "empty.html", "image/jpg").spoofed? }
+
+    it "rejects the file" do
+      assert spoofed?
+    end
+
+    it "logs info about the detected spoof" do
+      Paperclip.expects(:log).with('Content Type Spoof: Filename empty.html (image/jpg from Headers, ["text/html"] from Extension), content type discovered from file command: text/html. See documentation to allow this combination.')
```

Comment:
I think capture will read better. Also, by matching what we want to match we'll make the spec less brittle to log string changes. Do you think this would improve the spec?

```ruby
output = capture(:stdout) do
  run_arbitrary_code
end
assert output =~ /pattern/
```

Thank you!